xwiki-platform-core/xwiki-platform-oldcore/pom.xml

```diff
@@ -399,6 +399,21 @@
       <artifactId>xwiki-platform-template-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+
+    <!-- ExportAction -->
+
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-tree-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-index-tree-api</artifactId>
```

Comment:
You cannot add this dependency since it depends on oldcore. You will need to mock the implementation you need in it.

Reply:
Even when it's a test dependency?